ORION/unit-test/Makefile

```
@@ -0,0 +1,468 @@
+convert-all: generate-all-separate ## Convert all separate scenario files to bulk
+	@echo "$(GREEN)Converting all scenarios to bulk format...$(NC)"
+	@for file in $(GENERATED_DIR)/*.json; do \
+		base=$$(basename $$file .json); \
+		./json-to-bulk.py $$file $(GENERATED_DIR)/$$base.ndjson; \
+	done
+	@echo "$(GREEN)✓ Converted all scenario files to NDJSON$(NC)"
+
+##@ Step 3: Index to OpenSearch
+
+.PHONY: create-index
+create-index: ## Create index with correct mapping (iteration_id.keyword)
```

checkmake (0.3.2)

[warning] Target body for "create-index" exceeds allowed length of 5 lines (10).

(maxbodylength)